packages/relative-values/src/components/View/PlotView/index.tsx

```diff
@@ -1,33 +1,42 @@
-import { StyledTab } from "@/components/ui/StyledTab";
 import { ModelEvaluator } from "@/values/ModelEvaluator";
-import { SqLambda } from "@quri/squiggle-lang";
-import { FC } from "react";
+import clsx from "clsx";
+import { FC, PropsWithChildren } from "react";
 import { ClusterFilter } from "../ClusterFilter";
 import { ForcePlot } from "./ForcePlot";
 import { ValueAndUncertaintyPlot } from "./ValueAndUncertaintyPlot";
 
+const Section: FC<PropsWithChildren<{ title: string; border?: boolean }>> = ({
+  title,
+  border,
+  children,
+}) => (
+  <div className="flex flex-col gap-2">
+    <header className="text-slate-800 font-bold text-center">{title}</header>
+    <div className={clsx(border && "border border-slate-100 p-1")}>
+      {children}
+    </div>
+  </div>
+);
+
 export const PlotView: FC<{
   model: ModelEvaluator;
 }> = ({ model }) => {
   return (
-    <div className="flex gap-8 max-w-6xl mx-auto">
-      <div>
-        <StyledTab.Group>
-          <StyledTab.List>
-            <StyledTab name="Value and Uncertainty" icon={() => <div />} />
-            <StyledTab name="Force" icon={() => <div />} />
-          </StyledTab.List>
-          <StyledTab.Panels>
-            <StyledTab.Panel>
-              <ValueAndUncertaintyPlot model={model} />
-            </StyledTab.Panel>
-            <StyledTab.Panel>
-              <ForcePlot model={model} />
-            </StyledTab.Panel>
-          </StyledTab.Panels>
-        </StyledTab.Group>
-      </div>
-      <ClusterFilter axis="rows" />
+    <div
+      className="grid gap-2 max-w-6xl mx-auto"
```

Comment:
minor note, but I could see going full width for these, once the plots are variable width.

Reply:
Plots are variable width (I did this yesterday, as a part of improving the `useCanvas` hook, it now always measures the container and adapts; try resizing the window and see).